dora/shell/src/main/java/alluxio/cli/fs/command/MountCommand.java

```diff
@@ -77,14 +72,7 @@ public final class MountCommand extends AbstractFileSystemCommand {
    */
   public MountCommand(FileSystemContext fsContext) {
     super(fsContext);
-    AlluxioProperties properties = fsContext.getClusterConf().copyProperties();
-    properties.set(PropertyKey.DORA_ENABLED, false);
-    AlluxioConfiguration config = new InstancedConfiguration(properties);
-    mFileSystem = FileSystem.Factory.create(fsContext,
-        FileSystemOptions.Builder.fromConf(config)
-            .setUfsFallbackEnabled(false)
-            .build());
-    assert (mFileSystem instanceof BaseFileSystem);
+    mFileSystem = new BaseFileSystem(fsContext);
```

Comment:
nit: yimin made similar change so better unify the creation
```suggestion
    mFileSystem = FileSystem.Factory.createLegacy(fsContext);
```

Reply:
What's the point of the `createLegacy` function if it just redirects to the `BaseFileSystem` constructor?

Reply:
My guess is we want to have a reference place to see what commands( talking to the master) would be kept after clean up but I'm not sure.